docs/source/ui/pages/run_management.md

```diff
@@ -189,3 +199,155 @@ Use the `Settings` tab to delete or archive the single run
 
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Archive card" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/archive_card.png">
 <img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Unarchive card" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/unarchive_card.png">
+
+## Experiment page
+
+The experiment page provides overall information about a single experiment and helps to explore the runs attached to the experiment. 
```

Comment:
I wouldn't use the word "attached to experiment", maybe use "associated with the experiment" instead?